### ISSUE TITLE
Question - animation on Android < 12

### ISSUE THREAD

**Issue by vrublevskys** [vcaen/splashscreen-sample#2]
Hello! Is it possible to play AVD animation on pre 12 using splash screen API?

**Comment by vcaen** [comment]
No, unfortunately it's not possible, but you can use setOnExitAnimationListener to replace the still icon with an animated one manually, but it will increase your launch time
